flycheck.el

```diff
@@ -6368,11 +6368,32 @@ See Info Node `(elisp)Byte Compilation'."
             (princ (buffer-substring-no-properties (point-min) (point-max)))
             (kill-buffer)))))))
 
+(defconst flycheck-emacs-lisp-checkdoc-variables
+  '(checkdoc-symbol-words
+    checkdoc-arguments-in-order-flag
+    checkdoc-force-history-flag
+    checkdoc-permit-comma-termination-flag
+    checkdoc-force-docstrings-flag
+    checkdoc-package-keywords-flag
+    checkdoc-spellcheck-documentation-flag
+    checkdoc-verb-check-experimental-flag)
```

Comment:
Actually, I'm not sure that will work :/ It is quite possible for checkdoc to not be loaded but for a given checkdoc variable to be set, and in that case we still want to propagate the variable, right?
I don't think we can find the list of checkdoc variables without loading the package. We don't want to do that in the parent process, right?